flutter-idea/src/io/flutter/devtools/AbstractDevToolsViewFactory.java

```diff
@@ -98,19 +103,36 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
     }
 
     // Final case:
+    loadDevToolsInEmbeddedBrowser(project, toolWindow, flutterSdkVersion);
+
+    // Finally, listen for the panel to be reopened and potentially reload DevTools.
+    maybeReloadDevToolsWhenVisible(project, toolWindow, flutterSdkVersion);
+  }
+
+  private void loadDevToolsInEmbeddedBrowser(@NotNull Project project,
+                                             @NotNull ToolWindow toolWindow,
+                                             @NotNull FlutterSdkVersion flutterSdkVersion) {
     AsyncUtils.whenCompleteUiThread(
       DevToolsService.getInstance(project).getDevToolsInstance(),
       (instance, error) -> {
+        viewUtils.presentLabel(toolWindow, "Loading " + getToolWindowTitle() + "...");
+
         // Skip displaying if the project has been closed.
```

Comment:
rewrite this comment

Reply:
Oops autosubmit beat me to it! I will open a follow up PR monday